hw/top_darjeeling/top_darjeeling.core

```diff
@@ -87,10 +87,6 @@ filesets:
       # common waivers
       - lowrisc:lint:common
       - lowrisc:lint:comportable
-    files:
-      - lint/top_darjeeling.vbw
```

Comment:
Shouldn't we bring [this file over from `integrated_dev`](https://github.com/lowRISC/opentitan/blob/integrated_dev/hw/top_darjeeling/lint/top_darjeeling.vbw) rather than removing it here?